ServerModulesArrayApi.cfg

```diff
@@ -18,7 +18,7 @@ In1dMsg
 IndexingMsg
 JoinEqWithDTMsg
 KExtremeMsg
-LinalgMsg
+# LinalgMsg
```

Comment:
Intentionally commented out?

Reply:
Yeah, that module is an optional part of the array api and it's not finished yet, so I figured I'd remove it for now to speed up compilation times.